### DISCUSSION TITLE
How to add API Operation Label?

### DISCUSSION THREAD

**Discussion by Lzyct** [fuma-nama/fumadocs#864] [Q&A] [answered]
Hi, I see in the fumadocs documentation, there is like badges `PUT`,`POST`, etc 
How to add that to the side menu?

![image](https://github.com/user-attachments/assets/b539a629-c0b5-442b-bbe5-682a88427040)

This is my current implementation
![image](https://github.com/user-attachments/assets/eb690829-3859-4eb4-88a3-5acf158a7829)

Thank you

**Comment by fuma-nama** [comment, marked as answer]
It's in the setup guide:

https://fumadocs.vercel.app/docs/ui/openapi#generate-page-tree
